testsuite/features/secondary/min_ssh_push.feature

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Register a salt system to be managed via SSH push
+
+  Scenario: Delete the salt minion for ssh-reverse bootstrap
```

Comment:
what is "ssh-reverse" ? the name of a program?

If it's just "SSH reverse", then no dash, and use capital letters for protocol name.